### DISCUSSION TITLE
Unable to create Service Connection Error: Could not create a Personal Access Token for GitHub. Check your credentials and try again

### DISCUSSION THREAD

**Discussion by phoenixtekk** [community/community#37891] [New to GitHub]
Hello, Can someone please help me. I am getting the error below when trying to create a service connection. I know my password is correct. What could be wrong? 

**Could not create a Personal Access Token for GitHub. Check your credentials and try again.**

C:\Users\lacy>az pipelines create --name "First-Java.CI" --repository "https://github.com/phoenixtekk/pipelines-java" --branch "main"
Which service connection do you want to use to communicate with GitHub?
Please enter a choice [Default choice(1)]: Create new GitHub service connection

We need to create a Personal Access Token to communicate with GitHub. A new PAT with scopes (admin:repo_hook, repo, user) will be created.
You can set the PAT in the environment variable (AZURE_DEVOPS_EXT_GITHUB_PAT) to avoid getting prompted.
Enter your GitHub username (leave blank for using already generated PAT): phoenixtekk

Enter your GitHub password:
Confirm Enter your GitHub password:

**Could not create a Personal Access Token for GitHub. Check your credentials and try again.**

**Comment by ossanna16** [comment]
Hi there @phoenixtekk and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by queenofcorgis** [comment]
Hey there! Have you checked out our [docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) on this?
